src/constants/index.ts

```diff
@@ -1,12 +1,18 @@
 import { v4 as uuidv4 } from 'uuid';
 
-const VERSION = '2.0';
+const VERSION = '2.1';
```

Comment:
@flor-master are we going to allow a user to use the module with the default version?